Fabric.Authorization.AccessControl/src/app/modules/access-control/grain-list/grain-list.component.ts

```diff
@@ -155,12 +154,16 @@ export class GrainListComponent implements OnInit {
   }
 
   setSelectedNode(grain: string, securableItem?: string) {
+    const location = window.location;
+    const url = `${location.protocol}//${location.host}/client/access-control`;
```

Comment:
will host have the port number associated with it?  just making sure its parsing correctly

Reply:
Yes `host` includes the port. I'll try to pull it from config.